### DISCUSSION TITLE
salve ho fatto una popup card solo che quando la vado ad aprire mi si vede tutto blurrato

### DISCUSSION THREAD

**Discussion by leonardo20022002** [indomus/forum#3214] [GENERALE - Discussioni generiche]
![Screenshot_2](https://github.com/user-attachments/assets/e7c83396-7ed3-432f-9665-0ec00d291497)

questo è il primo codice 
type: horizontal-stack
cards:
  - type: custom:bubble-card
    hash: '#cat'
    name: statistiche
    icon: mdi:battery
    1_link: '#cat'
    1_name: statistiche sensori
    1_icon: mdi:battery
    1_entity: person.leonardo
    card_type: horizontal-buttons-stack
    margin: 5px
    width_desktop: 300px
    rise_animation: false

secondo codice 

type: vertical-stack
cards:
  - type: custom:bubble-card
    card_type: pop-up
    hash: '#cat'
    button_type: switch
    name: batterie dispositivi
    icon: mdi:battery
  - type: custom:bubble-card
    card_type: separator
    name: batteria sensori pir
    icon: mdi:battery
  - type: entities
    entities:
      - entity: sensor.0x00158d0007ee66b6_battery
      - entity: sensor.sensore_pir_ingresso_battery
  - type: custom:bubble-card
    card_type: separator
    name: 'barretia irrigatore '
    icon: mdi:sprinkler
  - type: entities
    entities:
      - entity: sensor.1f563e24004b1200_battery
  - type: custom:bubble-card
    card_type: separator
    name: batterie tende
    icon: mdi:roller-shade
  - type: entities
    entities:
      - entity: sensor.rollershutter_0001_batteria
      - entity: sensor.rollershutter_0002_batteria
      - entity: sensor.rollershutter_0003_batteria
      - entity: sensor.rollershutter_0004_batteria
  - type: custom:bubble-card
    card_type: separator
    name: batteria serratura smart
    icon: mdi:door
  - type: entities
    entities:
      - entity: sensor.lock_pro_3093_batteria

**Comment by Scialla** [comment]
correggi il post usando il markdown per cortesia
